plugins/bufbuild/connect-query/v0.1.0/buf.plugin.yaml

```diff
@@ -0,0 +1,21 @@
+version: v1
+name: buf.build/bufbuild/connect-query
+plugin_version: v0.1.0
+source_url: https://github.com/bufbuild/connect-query
+description: Generates client stubs for calling Connect services with TanStack Query.
+deps:
+  - plugin: buf.build/bufbuild/connect-web:v0.6.0
```

Comment:
Does it depend on `connect-web`, or the base types from `protobuf-es`. I have no idea tbh.

Reply:
You're too speedy - left a comment below. https://github.com/bufbuild/connect-query/blob/160b59f84ddb26126c47ee29ea8eb09ffe5d9d85/packages/connect-query/package.json#L39 shows there is a dependency on connect-web.

Reply:
Hmmm, I would have expected this to depend on `protobuf-es`. You're probably right, but I'd like to double-check if the generated code depends on connect-web generated code or protobuf-es generated code.

I don't understand why it needs the connect-web generated code, but I'm obviously not too familiar with this plugin.

Reply:
Yes, what @paul-sachs said is correct.  The _generated_ code itself does not depend on connect-web, but the functions that are used in the generated code depend on `Transport` at runtime, which is provided by `connect-web`.

Put another way, `connect-web` is a runtime dependency of a TypeScript web app that uses `connect-query`.  There is no way to use `connect-query` without `connect-web` because `connect-query` needs `Transport` from `connect-web` (and nothing else).

Reply:
@dimitropoulos I did notice that `connect-query` seems to have a [direct dependency](https://github.com/bufbuild/connect-query/blob/160b59f84ddb26126c47ee29ea8eb09ffe5d9d85/packages/connect-query/package.json#LL41C4-L41C4) on `connect-web` plugin though.  Perhaps that's unintentional?

Reply:
Updated this plugin to switch the dependency on generated code to bufbuild/es.

Reply:
At runtime, the one and only direct thing used from connect-web is `ConnectError` ([here](https://github.com/bufbuild/connect-query/blob/main/packages/connect-query/src/use-transport.tsx#L16)).  The rest of the direct dependencies are just types for Transport:
![Screenshot_20230126_125854](https://user-images.githubusercontent.com/15232461/214912895-e15dac64-383e-4f45-b499-7e2cd93350cf.png)